site/src/components/sections/Sections.vue

```diff
@@ -87,22 +87,27 @@
             ></font-awesome-icon>
             Full Section</span
           >
-
+          // Show the XL-full message if: // 1. The section is not already full
+          (avoids duplicate full section/full course) // - AND - // 2. The
+          course is an XL course and is out of XL seats.
           <span
             class="padding-left prerequisiteError"
             :class="{
-              hidden: section.rem <= 0 || !section.closed,
+              hidden:
+                section.xl_rem === undefined ||
+                section.xl_rem > 0 ||
+                section.rem <= 0,
             }"
             v-on:click.stop.prevent
             v-on:keyup.enter.stop.prevent
             @click="showSectionModal(section.crn)"
             @keyup.enter="showSectionModal(section.crn)"
           >
             <font-awesome-icon
-              :icon="['fas', 'exclamation-triangle']"
+              :icon="['fas', 'user-slash']"
             ></font-awesome-icon>
-            Closed
-          </span>
+            Full Course (No cross-list seats remaining)</span
```

Comment:
I think this should be changed to "Full Section", right?

Reply:
I did it intentionally as kinda a way of saying "the section may not be full but the course is full", I'm happy to change it to full section though

Reply:
Updated in latest push